assessments/FastBridge/seeds/assessmentReportingMethodDescriptors.csv

```diff
@@ -0,0 +1,19 @@
+codeValue,description,namespace,shortDescription,datatype,isComparisonPeriodMetric
```

Comment:
We typically make the codeValue a readable word because that is what people querying the data will use, so they won't know what 01 means, but would know what 'Composite Score' means

Reply:
Some of the comparison descriptors are exceeding the code value column length limit such as `GrowthPercentileAtDistrictLevelFromScreeningPeriod4toScreeningPeriod5`